user/caribou/module/src/dSiPMEventConverter.cc

```diff
@@ -210,7 +238,10 @@ bool dSiPMEvent2StdEventConverter::Converting(
   // Store frame begin and end in picoseconds
   d2->SetTimeBegin(frameStart);
   d2->SetTimeEnd(frameEnd);
-  d2->SetTriggerN(trigger_id_fpga);
+  d2->SetTriggerN(fpgadata.trigger_id);
+
+  // Add tag with frame number for additional information
+  d2->SetTag("frame", m_frame[plane_id]);
```

Comment:
In principle, the `StandardPlane` class object generated in L126 already has the feature of storing multiple frames (implemented for M26), see https://github.com/eudaq/eudaq/blob/master/main/lib/core/include/eudaq/StandardPlane.hh#L117

I'm not sure how useful this is for us, but maybe have a look to at least check if it makes sense.

Reply:
Hm, apparently we can attach this to each pixel: https://github.com/eudaq/eudaq/blob/2b22f1472498ffd0cc8d16a4afea52b26c24342f/main/lib/core/include/eudaq/StandardPlane.hh#L54

I have to look a bit where we can get this back. The idea was that we might be able to use this in corry to filter for a tag, but in the end, we just added it directly in the convert via `frame_start` and `frame_stop`.

Reply:
Ok looking at this:
https://github.com/eudaq/eudaq/blob/2b22f1472498ffd0cc8d16a4afea52b26c24342f/main/lib/core/src/StandardPlane.cc#L288

I don't seem to understand what this actually does...

Reply:
It's fine, I also like the tag - useful info anyway. :)